### DISCUSSION TITLE
添加About页面

### DISCUSSION THREAD

**Discussion by dczzzzzdc** [ppoffice/hexo-theme-icarus#867] [Q&A] [answered]
我想添加一个about，然后我在public的文件夹里面新添加一个一个叫about的文件夹。但是每次我 hexo g -d 之后就不见了。请问怎么设置一下。

**Comment by DullSword** [comment]
https://hexo.io/zh-cn/docs/setup#source
资源文件夹是存放用户资源的地方。除 _posts 文件夹之外，开头命名为 _ (下划线)的文件 / 文件夹和隐藏的文件将会被忽略。Markdown 和 HTML 文件会被解析并放到 public 文件夹，而其他文件会被拷贝过去。

在source下新建个about.md，不知道是不是你想要的。

**Reply by dczzzzzdc** [reply]
谢谢你的建议，但我指的是博客上方
Home  Archives  Categories  Tags   About 的那个about页面
然后，我点进去的时候就提示 Cannot GET /about

**Comment by ginagigo123** [comment, marked as answer]
你好～
我也跟你一樣正在嘗試加上About Page
然後我跑去別人的Theme看到原來About頁面大概是這樣加的：
https://github.com/aircloud/hexo-theme-aircloud

使用
`hexo new page about`
去新增，裡頭應該都有寫

**Reply by dczzzzzdc** [reply]
已经解决了，谢谢你的留言。

**Reply by Lukrisum** [reply]
搞定了，谢谢你的留言 :D
